packages/data-point/src/reducer-helpers/map/map.test.js

```diff
@@ -0,0 +1,38 @@
+const { ReducerMap } = require("./map");
+const { Accumulator } = require("../../Accumulator");
+const { resolve } = require("../../resolve");
+
+describe("ReducerMap", () => {
+  const spec = value => value * 2;
+
+  describe("constructor", () => {
+    it("should set type to 'constant'", () => {
+      const reducer = new ReducerMap(spec);
+      expect(reducer.type).toEqual("map");
+    });
+
+    it("should create a reducer from spec and assign to mapReducer", () => {
+      const reducer = new ReducerMap(spec);
+      expect(reducer.mapReducer).toHaveProperty("type", "function");
+    });
+  });
+
+  describe("create", () => {
+    it("should create a new instance of ReducerMap", () => {
+      expect(ReducerMap.create(spec)).toBeInstanceOf(ReducerMap);
+    });
+  });
+
+  describe("resolve", () => {
+    it("should map reducers to each element on the provided array", async () => {
+      const reducer = new ReducerMap(spec);
```

Comment:
rename to "spec" to "multiplyByTwo"? then it's easier to understand the test without scrolling the see what "spec" refers to